.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js:
-  - 0.6
-  - 0.8
+  - "5.4.0"
```

Comment:
please revert this; it's both already done in master and not relevant to this PR